### PR TITLE
fix: don't exclude first line on clear

### PR DESCRIPTION
**Pull request by codebytere**
Closes https://github.com/electron/fiddle/issues/709.

```js
const lines = output
  .slice(Math.max(output.length - 1000, 1))
  .map(this.renderEntry);
```

excludes the first element, which we wouldn't want to do especially when clearing Fiddles as it would lead to single dropped lines. Correct that.

### CODE REVIEW

**Review by erickzhao — COMMENTED** [review]
Seems like a snapshot test is failing with this change 😅

**Comment by dsanders11** [comment]
@codebytere, I think this can be simplified to just `.slice(-1000)`? Should be functionally equivalent and a bit easier to read, IMO.

Probably better for a follow-up issue/PR, but is there a reason to do the 1000 line max in the display, rather than when storing it in the output array? Is there another use for the output array other than display? I have noticed before in cases where there's a lot of console output that Fiddle will get sluggish until I clear the console, presumably because that array is getting quite large and causing garbage collection to kick on constantly. If the array is only for display then the max lines logic could be moved to where lines of output are added, which would prevent that.

**Review by ckerr — APPROVED** [review]
:+1: on this line fixing the existing bug. I agree with @dsanders11 about `.slice(-1000)` as being a slightly nicer improvement

Also I like David's point about trimming the output array pre-display just to avoid excess memory use, but that seems like a bigger scope that might make more sense as a standalone issue/PR

File: src/renderer/components/output.tsx [hunk 1]
```diff
@@ -138,7 +138,7 @@ export class Output extends React.Component<CommandsProps> {
 
     // The last 1000 lines
     const lines = output
-      .slice(Math.max(output.length - 1000, 1))
+      .slice(Math.max(output.length - 1000, 0))
```
Comment (ckerr) [inline src/renderer/components/output.tsx:141]:
```suggestion
      .slice(-1000)
```

Suggested by @dsanders11

Comment (ckerr) [inline src/renderer/components/output.tsx:141]:
```suggestion
      .slice(-1000)
```

**Review by erickzhao — APPROVED** [review]

**Comment by erickzhao** [comment]
LGTM pending test fix

**Pull request merged**